pkgs/development/tools/build-managers/conan/default.nix

```diff
@@ -98,6 +108,5 @@ in newPython.pkgs.buildPythonApplication rec {
     description = "Decentralized and portable C/C++ package manager";
     license = licenses.mit;
     maintainers = with maintainers; [ HaoZeke ];
-    platforms = platforms.linux;
```

Comment:
`platforms = platforms.unix` would probably be appropriate, no?

Reply:
The code seems to only depend on python, so I think it's safe to assume that it runs on `python`'s platforms